Product/SoapUI_Test/RegressionSuite/Passthru/AuditLogging-Passthrough/QueryDocumentAuditLog.properties

```diff
@@ -26,7 +25,6 @@ ActiveParticipant.RoleIDCode.@displayName=Human Requestor
 <!--Source-->
 	
 ActiveParticipant_Source.@UserID=http://www.w3.org/2005/08/addressing/anonymous
-ActiveParticipant_Source.@AlternativeUserID=19233@192.168.1.2
 ActiveParticipant_Source.@UserName=Karl Skagerberg
 ActiveParticipant_Source.@UserIsRequestor=true
 ActiveParticipant_Source.@NetworkAccessPointID=192.168.1.2
```

Comment:
@alameluchidambaram : This PR is about AlternativeUserID assertions and other things that are mentioned in PR description. As far as I know, I have not touched any assertions related to NetworkAccessPointID assertion block. But, based on your comment I checked script and I do see assertions and check statements for NetworkAccessPointID.